W1-M6-object-oriented/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace classes
+
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            /// <summary> 

```

Comment:
This won't work

You cannot have XML Code comments within a function block, only outside